utils/http.go

```diff
@@ -27,7 +27,16 @@ type rootHandler struct {
 	actions []string
 	context context.Context
 	trust   signed.CryptoService
-	//cachePool redis.Pool
+}
+
+func parseImageName(r *http.Request) string {
+	vars := mux.Vars(r)
```

Comment:
Slept on it, I think with the updates to the URL routes switching from `*` to `+` in the regexes, this is OK because only the changefeed will be able to produce an empty image name.